src/test/java/com/vanniktech/dependency/graph/generator/DependencyGraphGeneratorPluginTest.kt

```diff
@@ -269,6 +272,13 @@ class DependencyGraphGeneratorPluginTest {
         ":app" -> ":lib2" ["style"="dotted"]
         ":lib1" -> ":lib"
         ":lib2" -> ":lib"
-        }""".trimIndent())
+        }""".trimIndent()
+
+    assertThat(File(testProjectDir.root, "build/reports/project-dependency-graph/project-dependency-graph.dot")).hasContent(projectDependencyGraph(testProjectDir.root.name))
+
+    // We don't want to assert the content of the image, just that it exists.
+    assertThat(File(testProjectDir.root, "app/build/reports/project-dependency-graph/project-dependency-graph.svg")).exists()
+
+    assertThat(File(testProjectDir.root, "app/build/reports/project-dependency-graph/project-dependency-graph.dot")).hasContent(projectDependencyGraph("app"))
```

Comment:
This will make sure the graph generated from sub project `app` is correct. If you rollback the implementation code, this test will be failed, as the original code didn't include the nested dependencies.